audio/abcde/Portfile

```diff
@@ -19,25 +20,29 @@ long_description        abcde is a frontend command-line utility (actually, a \
 homepage                https://abcde.einval.com/
 master_sites            ${homepage}download/
 
-checksums               rmd160  e98d53ec173ecec0386282f9249423af7f9190fe \
-                        sha256  e49c71d7ddcd312dcc819c3be203abd3d09d286500ee777cde434c7881962b39
+checksums               rmd160  f3a2d81e3d3454221cd88c4bbebb56dc0c7a260b \
+                        sha256  70ec6e06b791115fbe88dee313f58f691f9b559ee992f2af5ed64fe6ad2e55d7 \
+                        size    159563
+
+perl5.branches          5.26
 
 depends_lib             port:vorbis-tools \
                         port:lame \
                         port:flac \
                         port:cd-discid \
                         port:cdparanoia \
                         port:id3v2 \
-                        port:normalize
+                        port:normalize \
+                        port:p5-digest-sha \
+                        port:p5-musicbrainz-discid \
+                        port:p5-webservice-musicbrainz
```

Comment:
These `p5-*` ports aren't frozen at 5.26, so you should use `p${perl5.major}-` instead. See port `mosh` for an example.

Reply:
I have opened #1982 to address this.